### PR TITLE
Added PostgeSQL connectivity

### PR DESCRIPTION
**Pull request by aign**
Hello. I'd like to offer you some changes to this project. I've added PostgreSQL connectivity to this script, to help anyone who wold like to save data to PostgeSQL server. Original functionality stays intact. I have added one key '--pg' to use PostgeSQL instead of sqlite.

### CODE REVIEW

File: gyb.py [hunk 1]
```diff
@@ -64,6 +65,16 @@
...
+import psycopg2
+from psycopg2.extensions import AsIs
+
+db_config = {
+ "host": "localhost", 
+ "port": "5432", 
+ "dbname": "gyb",
+ "user": "postgres",
+ "password": "111"
+}
```
Comment (jay0lee) [inline gyb.py]: These values should not be hard-coded, they should be flags.

**Comment by jay0lee** [comment]
Very cool, can you allow sql server / user / pass to be specified by flags and offer some basic usage guide?

**Comment by jay0lee** [comment]
Also, what if I want to backup more than one user to a database server? How is your code distinguishing between users?

File: gyb.py (hunk 1 above)
Comment (aign) [inline gyb.py]: Ok, will fix it

**Comment by aign** [comment]
Script creates a schema in database for each username

**Pull request closed without merging**